### ISSUE TITLE
Edit OCRed text

### ISSUE THREAD

**Issue by testingsw1** [ciur/papermerge#466]
For me OCR result are not that good and sadly sometimes really bad. I wish just to have at least few important key words OCRed.
Is it possible to add option "Edit OCRed text" (button via GUI, under "View OCRed text", above Tags) so I can manually fix text?

**Comment by ciur** [comment]
This feature makes sense. However, "Editing the OCR text" steps over the feet of versioning concept.
Say you have a document X.pdf which you OCRed. Thus you have the version 0 (original) and version 1 with OCRed  (of poor quality i.e. some words were not detected properly) text.
If you decide to edit the OCRed text, then later OCRing will "discard you text editing"

**Comment by testingsw1** [comment]
Thank you! Of course I will be fine with this. I think most of us are having these documents as archives and there is not a lot of versioning. This feature will really help! I have scanner with Abbyy software that does great job on some documents - with this feature I can easily replace wrong text in papermerge. Can't wait for new version!T hank you again!

**Comment by ciur** [comment]
> Ps. Is is possible to delete not needed versions via console? I can get to correct document via manage.py shell --> Document.objects.get(id=XX), but I am not sure how to get (and delete) specific version.

Yes - just keep in mind that I've never tested what happens when you delete document version and there is no such "official feature" :)

In shell>
```
In [2]: from papermerge.core.models import Document
In [3]: doc = Document.objects.first()
In [4]: doc_version = doc.versions.get(number=2)
In [5]: doc_version
Out[5]: <DocumentVersion: id=f6298a48-a991-4b8a-a75c-5fde6d899c4b number=2>
In [6]: doc_version.delete()
Out[6]: (3, {'core.Page': 2, 'core.DocumentVersion': 1})
```
Where ``number=2`` is version number of document instance.
